Tests/Foundation/Tests/TestFileManager.swift

```diff
@@ -1755,6 +1755,55 @@ VIDEOS=StopgapVideos
         }
         #endif
     }
+
+    func test_windowsPaths() throws {
+        let fm = FileManager.default
+        let tmpPath = writableTestDirectoryURL.path
+        do {
+            try fm.createDirectory(atPath: tmpPath, withIntermediateDirectories: true, attributes: nil)
+        } catch {
+            XCTFail()
+        }
+        fm.changeCurrentDirectoryPath(tmpPath)
+        func checkPath(path: String) throws {
+            XCTAssertTrue(fm.createFile(atPath: path, contents: Data(), attributes: nil))
+            XCTAssertTrue(fm.fileExists(atPath: path))
+            try fm.removeItem(atPath: path)
+            XCTAssertFalse(fm.fileExists(atPath: path))
+
+            let url = URL(fileURLWithPath: path)
+            try "hello world".write(to: url, atomically: false, encoding: .utf8)
+            try fm.removeItem(atPath: path)
+            XCTAssertFalse(fm.fileExists(atPath: path))
+        }
+        let paths = [
+            // Absolute Paths
+            "\(tmpPath)\\testfile",       // C:/Users/...\testfile
+            "\(tmpPath)/testfile",        // C:/Users.../testfile
+            "\(tmpPath)/testfile",        // /Users/user/.../testfile
+            "\(tmpPath.first!):testfile", // C:testfile
```

Comment:
Btw, this fails when runner has no Administrator privileges. CI agents are usually elevated, but manual run as user would report failure.